shell/components/ResourceTable.vue

```diff
@@ -256,9 +256,7 @@ export default {
       const haveAllNamespace = this.$store.getters['haveAllNamespace'];
 
       return this.rows.filter((row) => {
-        if (this.currentProduct?.hideSystemResources && this.isNamespaced) {
```

Comment:
Same as with NameNsDescription comment, I don't think it makes sense to stop using `hideSystemResources` here when it is still used elsewhere.